webapp/pages/admin/users.vue

```diff
@@ -128,6 +138,7 @@ export default {
               id
               name
               slug
+              email
```

Comment:
At the moment, the backend does not allow anybody except the owner to see the `email` of the user. You will need to change permissions in the backend.

I think I would still feel safe, if only the owner *or* an admin user can see the email.

Reply:
Done, thanks @roschaefer